events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -465,13 +473,13 @@
                 });
             } else {
                 currentChoices = [];
-                currentChoices.push(triviaData[0].correct_answer);
+                currentChoices.push({"text": triviaData[0].correct_answer, "correct": true});
                 triviaData[0].incorrect_answers.forEach(function(choice) {
-                    currentChoices.push(choice);
+                    currentChoices.push({"text": choice, "correct": false});
```

Comment:
This code is a bit confusing on what it does. I'd 1 or 2 comments to explain what is happening.

Reply:
Commented :)